.github/workflows/deploy.yml

```diff
@@ -0,0 +1,48 @@
+name: Deploy
+on:
```

Comment:
Can we have the following triggers set?
For nightly, it should be deployed upon push on dev branch
For stable, it should be deployed upon push on tag "[v]?[0-9]+.[0-9]+.[0-9]+*"

Reply:
Oh both should only run after ci.yml succeeds,. refer to https://github.com/tlsnotary/tlsn/blob/dev/.github/workflows/cd.yml#L19-L27

Reply:
Done! https://github.com/ntampakas/tlsn/commit/e9776aa45d9adc35a597b4f24000bb08bb51ca79